test/tck/configure_feature_files.py

```diff
@@ -0,0 +1,30 @@
+import os
+import tarfile
+import urllib2
```

Comment:
You are probably right. This has currently been build only focusing on python 2.7 at the moment.